### DISCUSSION TITLE
chrome://flags#install-isolated-web-app-from-url

### DISCUSSION THREAD

**Discussion by MechaXYZ** [3kh0/ext-remover#966] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Maybe this flag could be used for an exploit?

**Comment by nathblade16** [comment]
I don't think so????

**Comment by helloIlovegimkit** [comment, marked as answer]
> Maybe this flag could be used for an exploit?

After reading about it, it doesn't seem like you can use this flag for an exploit as it needs to be signed with something it seems
